com.unity.cinemachine/Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -920,7 +920,7 @@ public bool IsLive(ICinemachineCamera vcam, bool dominantChildOnly = false)
         /// that is visible to my camera.  Culling Mask is used to test visibility.
         /// </summary>
         /// <returns>The highest-priority Enabled ICinemachineCamera that is in my visible layers.</returns>
-        internal protected virtual ICinemachineCamera TopCameraFromPriorityQueue()
+        internal virtual ICinemachineCamera TopCameraFromPriorityQueue()
```

Comment:
why not internal protected?

Reply:
If you add a protected access modifier to a public class it becomes part of the public API. Because if someone inherit from it they can access it. We would need a minor bump and not a patch bump to do this change.

I tried to run the API validation with it but didn't pass.
See
https://unity-ci.cds.internal.unity3d.com/job/32285233/logs
<img width="1041" alt="Screenshot 2023-12-13 at 4 08 48 PM" src="https://github.com/Unity-Technologies/com.unity.cinemachine/assets/10277745/fd7d2396-ac77-46f0-8674-b8b43445bc99">

Reply:
ok take out internal and protected - make it private